vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+- name: create minio user
```

Comment:
please spaces between each tasks.
This goes for all the other tasks down this main.yml